core/instrumented/instrument.go

```diff
@@ -138,14 +138,14 @@ func (ibs *BlockStore[I]) Add(ctx context.Context, rawBlock core.RawBlock[I]) (c
 
 // BlockSender is a BlockSender that records stats for events.
 type BlockSender[I core.BlockId] struct {
-	sender core.BlockSender[I]
-	stats  stats.Stats
+	blockSender core.BlockSender[I]
```

Comment:
To avoid conflating with status sender, again mostly for new comers to the code, but when you're juggling too many ideas at once...